File: src/main/java/com/itmo/microservices/users/entity/AppUser.java
package com.itmo.microservices.users.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class AppUser {

    @Id
    private String username;
    private String name;
    private String surname;
    private String email;
    private String password;
}

File: src/main/kotlin/com/itmo/microservices/demo/users/impl/entity/AppUser.kt
package com.itmo.microservices.demo.users.impl.entity

import java.util.*
import javax.persistence.Entity
import javax.persistence.Id

@Entity
class AppUser {

    @Id
    var id: UUID? = null
    var ipaddress: String? = null
    var name: String? = null
    var username: String? = null
    var email: String? = null
    var password: String? = null
    var phone: String? = null
    var lastBasketId: UUID? = null

    constructor()

    constructor(id: UUID?, ipaddress: String?, name: String?,  username: String?, email: String?, password: String?, phone: String?, lastBasketId: UUID?) {
        this.id = id
        this.ipaddress = ipaddress
        this.username = username
        this.name = name
        this.email = email
        this.password = password
        this.phone = phone
        this.lastBasketId = lastBasketId
    }

    constructor(id: UUID?) {
        this.id = id
    }




}